stable/kommander/values.yaml

```diff
@@ -238,7 +238,6 @@ kubecost:
         datasources:
           enabled: true
           defaultDatasourceEnabled: false
-          # dataSourceName: KubecostPrometheus
```

Comment:
just a leftover eh?

Reply:
hmm.. actually I'm gonna add this back (uncomment it). originally this field wasn't supported/respected but it is now and `KubecostPrometheus` is more descriptive than the default `Prometheus`

Reply:
:+1: